modules/programs/delta.nix

```diff
@@ -135,14 +135,21 @@ in
       })
 
       (lib.mkIf (cfg.enable && cfg.enableJujutsuIntegration) {
+        assertions = [
+          {
+            assertion = config.programs.jujutsu.enable or false;
+            message = "programs.delta.enableJujutsuIntegration requires programs.jujutsu.enable to be true";
+          }
+        ];
+
         programs.jujutsu.settings = {
-          merge-tools.delta.diff-expected-exit-codes = [
+          merge-tools.delta.diff-expected-exit-codes = lib.mkDefault [
             0
             1
           ];
```

Comment:
Given that lists merge, I don't see the need to be able to _override_ this value. JJ _will_ complain if Delta exits with 1 by default.